wrappers/saml/saml.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Created by odysseasg on 02/06/2018.
+ *
+ * Based on prototype by tacb0ss.
+ */
+
+
+const Wrapper = require('../wrapper-core').Wrapper;
+const saml2 = require('saml2-js');
```

Comment:
if required here it forces the presence of the module in the project...

The require("saml2-js") should be in the _ connect callback